opm/utility/ECLGraph.hpp

```diff
@@ -138,6 +136,9 @@ namespace Opm {
         /// strictly positive.
         std::vector<double> poreVolume() const;
 
+        /// Enum for indicating requested phase from the flux() method.
+        enum PhaseIndex { Aqua = 0, Liquid = 1, Vapour = 2 };
```

Comment:
Any reason **not** to make this an `enum class` if we don't have to be source-level compatible with opm-core?